stations.csv

```diff
@@ -4948,13 +4948,13 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 6548;Algeciras;algeciras;7155020;;36.12672;-5.450995;;ES;Europe/Madrid;f;f;f;t;f;f;ESAFR;;f;;f;7100136;f;eykjeb;t;ESAEI;t;23571;t;;f;;f;;f;;f;;f;;;;f;;f;55020;t;;f;ESAFR;f;;f;f;;;;;Algésiras;;;;;;;アルヘシラス;알헤시라스;;;Альхесирас;;;阿尔赫西拉斯;urn:trainline:public:nloc:csv6548;
 6549;Sagunt;sagunt;7165200;;39.675779;-0.271564;;ES;Europe/Madrid;f;f;f;t;f;f;ESAFS;;f;;f;7100055;f;ezpccb;t;;f;32391;t;;f;;f;;f;;f;;f;;;;f;;f;65200;t;;f;ESAFS;f;;f;f;;;;Sagunto;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6549;
 6550;Sabadell Sud;sabadell-sud;7178703;;41.528725;2.105183;27261;ES;Europe/Madrid;f;f;f;t;f;f;ESAFT;;f;;f;7178703;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;78703;t;;f;ESAFT;f;;f;f;;;;Sur;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6550;
-6551;Puigcerdà;puigcerda;7100291;;42.4316;1.92819;;ES;Europe/Madrid;f;f;f;t;f;f;ESAFU;;f;;f;;f;sp937u;t;ESPUI;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6551;
+6551;Puigcerdà;puigcerda;7100291;;42.4316;1.92819;;ES;Europe/Madrid;f;f;f;f;f;f;ESAFU;;f;;f;;f;sp937u;t;ESPUI;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6551;
```

Comment:
Why have you changed is suggestible to false on this station?

Reply:
Not sure how that happened but its fixed now

Reply:
I think the same comment applies to other stations - for example, `Ripoll`, `Pineda de Mar`, and `Santa Susanna`. Perhaps others also.